lms/js_config_types.py

```diff
@@ -74,13 +74,24 @@ class APIStudents(TypedDict):
 
 class DashboardRoutes(TypedDict):
     assignment: str
+    """Fetch a single assigment by ID"""
+
     assignment_stats: str
```

Comment:
I don't want to change the names of these right now to allow having to coordinate the FE but I reckon we'd prefer "metrics" over "stats".

Reply:
I think the "metrics" terminology was mentioned at some point, but I think it's ok to keep "stats". They are synonyms, at least in this context.